src/fu-util.c

```diff
@@ -2004,9 +2176,33 @@ fu_util_modify_remote (FuUtilPrivate *priv, gchar **values, GError **error)
 				     "Invalid arguments");
 		return FALSE;
 	}
-	return fwupd_client_modify_remote (priv->client,
-					   values[0], values[1], values[2],
-					   NULL, error);
+	return fu_util_modify_remote (priv, values[0], values[1], values[2], error);
+}
+
+static gboolean
+fu_util_remote_enable (FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
Since you have introduced this new function you should update `fu_util_download_metadata_enable_lvfs`  to use it too.

Reply:
I did :)

Reply:
Huh, I mean specifically line 1429:
`+	if (!fu_util_modify_remote (priv, "lvfs", "Enabled", "true", error))`
Changing that to use `fu_util_remote_enable`

Reply:
Ahh, I see what you mean. I didn't use that as `fu_util_remote_enable` is a command handler, and it's not hugely convenient to pass a single item gchar** in C. It all works out in the end.